features/cart/shopping_cart/seeing_detailed_information_about_cart.feature

```diff
@@ -0,0 +1,22 @@
+@shopping_cart
+Feature: Seeing detailed information of cart
+    In order to see detailed information of my cart
+    As a Visitor
+    I want to be able to see those information of my cart
```

Comment:
`In order to see (...) information, (...), I want to be able to see those information` :( I believe the goal could be described better.

Reply:
I hope that newest changes with my limited humanistic possibilities would fulfill your desires 😄